src/core/thread/mle.cpp

```diff
@@ -295,10 +296,16 @@ ThreadError Mle::Restore(void)
 
     if (networkInfo.mDeviceState == kDeviceStateChild)
     {
-        length = sizeof(mParent);
+        length = sizeof(parentInfo);
         SuccessOrExit(error = otPlatSettingsGet(mNetif.GetInstance(), kKeyParentInfo, 0,
-                                                reinterpret_cast<uint8_t *>(&mParent), &length));
-        VerifyOrExit(length == sizeof(mParent), error = kThreadError_NotFound);
+                                                reinterpret_cast<uint8_t *>(&parentInfo), &length));
+        VerifyOrExit(length == sizeof(parentInfo), error = kThreadError_NotFound);
```

Comment:
Updated, thanks (note: Need to rebase to #1572 which includes otPlatSettingsGet() change).